tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -54,10 +54,6 @@ else
 	)
 fi
 
-if [ ${KATA_HYPERVISOR} == "qemu-sev" ]; then
-	exit 0
-fi
-
```

Comment:
As mentioned in the PR raised by @fitzthum, let's drop this part and re-add it after this PR gets merged, as the artefacts are not being taken into consideration yet due to how `on: pull_request_target` works. :-/